packages/flutter_tools/templates/create/ios.tmpl/Runner/AppDelegate.m

```diff
@@ -5,42 +5,7 @@ @implementation AppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
   [GeneratedPluginRegistrant registerWithRegistry:self];
-  // Override point for customization after application launch.
```

Comment:
We might want to keep this comment.

The main reason for having the other (at that point, empty) methods was to show where to add code (plus, they're from Xcode's new iOS project template...).